tests/documentation/DocGrammarBearTest.py

```diff
@@ -0,0 +1,127 @@
+from queue import Queue
+import unittest
+import shutil
+
+from coalib.results.Diff import Diff
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import execute_bear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from bears.documentation.DocGrammarBear import DocGrammarBear
+from string import Template
+
+
+try:
+    import language_check
+    language_check
+except ImportError as err:
+    raise SkipTest(str(err))
+
+
+def make_docstring(main_desc: str='',
+                   param_desc: str='',
+                   return_desc: str=''):
+    docstring = Template('"""\n' +
+                         '$main_description' +
+                         '\n' +
+                         ':param xyz: $param_description' +
+                         ':return: $return_description' +
+                         '"""\n')
+    return docstring.substitute(main_description=main_desc,
+                                param_description=param_desc,
+                                return_description=return_desc
+                                ).splitlines(True)
+
+
+def test(test_file, expected_file, optional_setting=None):
```

Comment:
we are giving a `docstring/file` data(explicit tests).
Maybe I should rename them to `test_data`, `expected_data`